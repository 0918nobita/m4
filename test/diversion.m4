include(syntax.m4)dnl
divert(1)dnl
a divnum
foo
divert(0)dnl
b divnum
bar
undivert(1)dnl
c divnum
baz
