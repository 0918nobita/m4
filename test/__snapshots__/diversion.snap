b 0
bar
a 1
foo
c 0
baz
